atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -87,7 +113,7 @@ def run(self):
             if not build:
                 break
```

Comment:
Right, added new tests parameter - `exists`. Now `getBuild` would return more realistic results - `True` if 'major' part of the releases matches (and if `exists` is set a full release string is matched)